README.md

```diff
@@ -1 +1,70 @@
-Interface mockup
\ No newline at end of file
+# Microscopium User Interface
+
+Prerequisites
+-------------
+
+* Node.js > v0.10.35
+
+Install Instructions
+--------------------
+
+```
+$ npm install
+$ bower install
+```
+
+Run dev version locally with
+```
+grunt
```

Comment:
Still need to add these:

    ```console
    $ grunt
    ```
And add the `console` language keyword to the npm/bower lines.